File: BookMyShowWebAPI/ClientApp/src/app/Model/booking.ts

export class Booking {
    theaterId:number;
    showId:number;
    ticketPrice:number;
    seatNumber:number;
    bookingDate:Date;

    constructor( theaterId:number,showId:number,ticketPrice:number,seatNumber:number,bookingDate:Date){
        this.theaterId = theaterId;
        this.showId = showId;
        this.ticketPrice = ticketPrice;
        this.seatNumber = seatNumber;
        this.bookingDate = bookingDate;
    }
}

File: BookMyShow.Models/Booking.cs
using System;

namespace BookMyShowWebAPI.Models
{
    public class Booking
    {
        public int TheaterId { get; set; }
        public int ShowId { get; set; }
        public int TicketPrice { get; set; }
        public int SeatNumber { get; set; }
        public DateTime Date { get; set; }
    }
}